pom.xml

```diff
@@ -58,8 +58,8 @@
     <jee.version>7</jee.version>
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
     <avro.version>1.7.7</avro.version>
-    <cdap.version>6.0.0-SNAPSHOT</cdap.version>
-    <cdap.plugin.version>2.2.0-SNAPSHOT</cdap.plugin.version>
+    <cdap.version>6.1.0-SNAPSHOT</cdap.version>
```

Comment:
why change this?
This change doesn't break anything, but it should remain 6.0.0-SNAPSHOT, since this is the branch used to build cdap 6.0.

Reply:
Ah, I think its because you started your PR branch from develop.

Reply:
Undoing the change in https://github.com/data-integrations/google-cloud/pull/104.